src/java.base/share/classes/java/util/function/BiConsumer.java

```diff
@@ -72,4 +72,37 @@ default BiConsumer<T, U> andThen(BiConsumer<? super T, ? super U> after) {
             after.accept(l, r);
         };
     }
+
+    /**
+     * {@return a representation of the provided {@code uncaptured} lambda or method reference
+     * in the form of a {@code BiConsumer}}
+     * <p>
+     * This method is useful in cases where there is an ambiguity in a lambda or method reference
+     * or when using composition or fluent coding as shown in this example:
+     * {@snippet :
+     * void toConsole(long id, String message) {
+     *      System.out.format("%d = %s%n", id, message);
+     * }
+     *
+     * void toLogger(long id, String message) {
+     *      LOGGER.info(String.format("%d = %s", id, message));
+     * }
+     *
+     * // Capturing
+     * var con = BiConsumer.of(this::toConsole); // BiConsumer<Long, String>
+     *
+     *  // Fluent composition
+     * var composed = BiConsumer.of(this::toConsole)
+     *                    .andThen(this::toLogger);  // BiConsumer<Long, String>
+     *
+     * }
+     *
+     * @param uncaptured to capture
+     * @param <T> the type of the first argument to the operation
+     * @param <U> the type of the second argument to the operation
+     */
+    static <T, U> BiConsumer<T, U> of(BiConsumer<T, U> uncaptured) {
```

Comment:
```suggestion
    static <T, U> BiConsumer<T, U> of(BiConsumer<? super T, ? super U> uncaptured) {
```

Reply:
@ExE-Boss This and all other "enhancements" hardly compile ;-)